packages/plugin-aws-lambda/test/session.test.ts

```diff
@@ -0,0 +1,124 @@
+import Client from '@bugsnag/core/client'
+import Session from '@bugsnag/core/session'
+import type { Plugin } from '@bugsnag/core'
+import BugsnagPluginAwsLambdaSession from '../src/session'
+
+describe('plugin: aws lambda sessions', () => {
+  describe('without server plugin', () => {
+    it('can start a session', () => {
+      const client = createClient()
+
+      expect(client._session).toBeNull()
+
+      const shouldNotBeCloned = client.startSession()
+
+      expect(shouldNotBeCloned).toBe(client)
+      expect(client._session).toBeInstanceOf(Session)
+      // @ts-ignore
+      expect(client._session.toJSON()).toEqual({
+        events: {
+          handled: 0,
+          unhandled: 0
+        },
+        id: expect.any(String),
+        startedAt: expect.any(Date)
+      })
+    })
+
+    it('can pause and resume a session', () => {
+      const client = createClient()
+
+      expect(client._session).toBeNull()
+
+      const shouldNotBeCloned = client.startSession()
+
+      expect(shouldNotBeCloned).toBe(client)
+      expect(client._session).toBeInstanceOf(Session)
+      const expectedSession = client._session
+
+      client.pauseSession()
+
+      expect(client._session).toBeNull()
+
+      // resumeSession should not clone either
+      const resumeSessionClient = client.resumeSession()
+
+      expect(client._session).toBe(expectedSession)
+      expect(resumeSessionClient).toBe(client)
+    })
+
+    it('does not clone the client with resumeSession if there is no active session', () => {
+      const client = createClient()
+      const cloned = client.resumeSession()
+
+      expect(cloned).toBe(client)
+      expect(client._session).toBeInstanceOf(Session)
+    })
+  })
+
+  ;['express', 'koa', 'restify'].forEach(serverPlugin => {
```

Comment:
```suggestion
  ['express', 'koa', 'restify'].forEach(serverPlugin => {
```
?

Reply:
You can't start a line with `[` without a semicolon before it, otherwise it gets interpreted like this:

```js
describe('', function () {})['express', ...].forEach(...)
```

but in this case it can be an `it.each` and sidestep that problem